data-raw/process_data_set_irene.R

```diff
@@ -0,0 +1,74 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+castle_solutions <- read_csv("data-raw/riddler-castles/castle-solutions.csv")
+
+castle_solutions <- castle_solutions %>% #clean the names
+  rename(strategy = `Why did you choose your troop deployment?`) %>%
+  clean_names()
+
+castle_solutions$user <- seq.int(nrow(castle_solutions)) #create user column
+  
+#gather the data to tidy form
+castle_solutions <- gather(castle_solutions, key = "castle", value = "number_of_soldiers", 
+                           contains("castle_"), -strategy, -user)
+
+#reorder
+castle_solutions <- castle_solutions[c("user", "castle", "number_of_soldiers", "strategy")]
```

Comment:
As per @rudeboybert 's preferences #61 , please use a `tidyverse` style. So here you would replace the `[` with a call to `select()`.